packages/core/test/types.test.ts

```diff
@@ -241,25 +241,25 @@ describe('Raise events', () => {
           on: {
             DECIDE: [
               {
-                actions: raise<GreetingContext, { type: 'ALOHA' }>({
+                actions: raise({
                   type: 'ALOHA'
-                }),
+                }) as any /* TODO: FIX */,
```

Comment:
Hm, I've wanted to look into it and I've checked out your branch locally but after removing those casts and comments I couldn't see any TS errors. Could you recheck on your machine? That being said - inferred generics are off and I don't understand why the inferred type is assignable here to the expected one 😭

Reply:
Hm, I'll keep the comment in here for now.